### PR TITLE
[Figgy] Use health.json for health checks

### LINKED ISSUE #5103: [Figgy] Use health.json for health checks
### What maintenance needs to be done?

When Figgy's health checks are failing (health.json), it should not serve that application to the public.

### Level of urgency
- [ ] High
- [x] Moderate
- [ ] Low

### Why is this maintenance needed?

This will make it easier and safer to deploy to only some of the machines, because we won't have to adjust the nginx configuration to take things off the load balancer just in case it breaks.

### Acceptance criteria
 - [ ] When one node is broken it does not get traffic diverted to it.
 - [ ] It checks `/health.json`

### Implementation notes, if any

Nginx Documentation: https://docs.nginx.com/nginx/admin-guide/load-balancer/http-health-check/
Existing health check configuration: https://github.com/pulibrary/princeton_ansible/blob/main/roles/nginxplus/files/conf/http/figgy-staging.conf

### First Step

Add the health check URL to the staging configuration.

### PR DESCRIPTION
**Pull request by tpendragon**
Closes #5103

### CODE REVIEW

**Review by kayiwa — APPROVED** [review]
OK! Thanks @tpendragon

File: roles/nginxplus/files/conf/http/figgy-staging.conf [hunk 1]
```diff
@@ -50,7 +50,7 @@ server {
         proxy_cache_convert_head off;
         proxy_cache_methods GET HEAD;
         proxy_cache_key $scheme$request_method$proxy_host$request_uri;
-        health_check interval=10 fails=3 passes=2;
+        health_check uri=health.json interval=10 fails=3 passes=2;
```
Comment (maxkadel) [inline roles/nginxplus/files/conf/http/figgy-staging.conf:53]: Does it matter that this one doesn't have the slash in front of `health.json`?

**Review by maxkadel — CHANGES_REQUESTED** [review]
Not sure if it works without the slash?

File: roles/nginxplus/files/conf/http/figgy-staging.conf (hunk 1 above)
Comment (tpendragon) [reply roles/nginxplus/files/conf/http/figgy-staging.conf]: Good catch, fixed.

**Review by maxkadel — APPROVED** [review]
Thanks @tpendragon 🐲

**Pull request merged**